bot/api.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, loop: asyncio.AbstractEventLoop, **kwargs):
 
     @staticmethod
     def _url_for(endpoint: str) -> str:
-        return f"{URLs.site_schema}{URLs.site_api}/{quote_url(endpoint)}"
+        return urlparse(f"{URLs.site_api}/{quote_url(endpoint)}", "http").geturl()
```

Comment:
Not sure why this results in three forward slashes but at least browsers don't seem to care...